Common/WALinuxAgent-2.0.16/waagent

```diff
@@ -2795,7 +2795,7 @@ class Logger(object):
         """
         return (counter < 10) or ((counter < 100) and ((counter % 10) == 0)) or ((counter % 100) == 0)
 
-    def LogToFile(self, message):
+    def WriteToFile(self, message):
```

Comment:
This file is shared by other extensions in this repo.  It isn't clear if these changes are self-contained or not.  Please double check that no other extensions are using these methods.

Reply:
I just double checked. The previous "LogToFile()" method was only used by CSE and waagent.py (at least these are the only references I found from the project).